test/IDE/print_usrs.swift

```diff
@@ -149,13 +149,91 @@ class Observers {
   }
 }
 
-// CHECK: [[@LINE+2]]:7 s:14swift_ide_test10ObjCClass1C{{$}}
+// CHECK: [[@LINE+2]]:7 c:objc(cs)ObjCClass1{{$}}
 @objc
 class ObjCClass1 {
-  // CHECK: [[@LINE+1]]:8 s:14swift_ide_test10ObjCClass1C13instanceFunc1ySiF{{$}}
-  func instanceFunc1(_ a: Int) {}
-  // CHECK: [[@LINE+1]]:14 s:14swift_ide_test10ObjCClass1C11staticFunc1ySiFZ{{$}}
+  // CHECK: [[@LINE+1]]:7 c:objc(cs)ObjCClass1(py)instanceVar{{$}}
+  var instanceVar: Int = 1
+  // CHECK: [[@LINE+1]]:14 c:objc(cs)ObjCClass1(cpy)typeVar{{$}}
+  static var typeVar: Int = 1
+
+  // CHECK: [[@LINE+1]]:7 c:objc(cs)ObjCClass1(py)computed{{$}}
+  var computed: Int {
+    // CHECK: [[@LINE+1]]:5 c:objc(cs)ObjCClass1(im)computed{{$}}
+    get { return 1}
+    // CHECK: [[@LINE+1]]:5 c:objc(cs)ObjCClass1(im)setComputed:{{$}}
+    set {}
+  }
+
+  // CHECK: [[@LINE+1]]:14 c:objc(cs)ObjCClass1(cpy)typeComputed{{$}}
+  static var typeComputed: Int {
```

Comment:
It does at the moment, yeah:
SWIFT_CLASS_PROPERTY(@property (nonatomic, class) NSInteger typeComputed;)
+ (NSInteger)typeComputed SWIFT_WARN_UNUSED_RESULT;
+ (void)setTypeComputed:(NSInteger)newValue;